### PR TITLE
Template

### PR DESCRIPTION
**Pull request by HiredMussel**
Creation of template completed. Upon review, can proceed to creating sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	<title>Alex Musselwhite</title>
+</head>
+<body>
+	<!-- the "home" section contains no content, but exists so that a link may be provided back to the very top of the page -->
+	<header id="home">
+	</header>
+	
+	<!-- Navigation Bar at the top of page. Links point to "sections" further down page in "main" -->
+	<nav>
+		<p class=myName>Alex Musselwhite</p>
```
Comment (Hannahrey) [inline index.html]: missing "" around myName

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	<div class="navbarDummy">
+		 
+	</div>
+	
+	<section class="callout">
+		<!-- container class added to allow for alignment by CSS -->
+		<div class="container">
+			<h1>Hi There!</h1>
+			<div class="container2">
+				<p> Nice to meet you! <br>
```
Comment (Hannahrey) [inline index.html]: missing "/" in self closing br tag. not sure if this matters though!

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+/* Navigation Bar */
+	/* 	The Navigation bar is formatted as grey text on a black background with a subtle "smoked glass" transparency effect. Mousing over 
+		the links will cause the text on them to turn white */
+	nav {
+	/* 	navbar's position is fixed on the page - needs to always be attached to top. This requires manual resizing to display correctly */
+		position: fixed;
+		top: 0;
+		z-index: 5;
+	
+		height: auto;
```
Comment (Hannahrey) [inline styles.css]: is this correct?

File: index.html (hunk 1 above)
Comment (HiredMussel) [reply index.html]: Thanks - added it in

File: index.html (hunk 2 above)
Comment (HiredMussel) [reply index.html]: Looked at a style guide linked from the project page on the Mayden github, which actively advised against this. I've therefore not been including that.

File: styles.css (hunk 3 above)
Comment (HiredMussel) [reply styles.css]: This should allow the navbar to set its own height based on that of its parent elements. I added it to resolve a bug with the behaviour of the navbar after first giving it position: fixed.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+		.callout {
+			font-size: 0.4em;
+		}
+	}
+
+/* Main Section */
+
+	main .container {
+		background-color: #EEEEEE;
+		border: 2px solid #000000;
```
Comment (Hannahrey) [inline styles.css]: you've used px here, and em everywhere else

Comment (HiredMussel) [reply styles.css]: Setting a border in em would mean that small text caused the border to disappear. Using px is safe even on very small screens

**Pull request merged**